specs/ClientCertificate.md

```diff
@@ -0,0 +1,500 @@
+      COREWEBVIEW2_CLIENT_CERTIFICATE_KIND* certificateKind);
+}
+
+/// A list of distinguished certificate authorities.
+[uuid(c87bb63e-a3f4-11eb-bcbc-0242ac130002), object, pointer_default(unique)]
+interface ICoreWebView2CertificateAuthorityList : IUnknown {
+  /// The number of certificate authorities contained in
+  /// ICoreWebView2CertificateAuthorityList.
+  [propget] HRESULT Count([out, retval] UINT* count);
+
+  /// Gets the certificate authority at the given index.
+  HRESULT GetValueAtIndex([in] UINT index, [out, retval] LPWSTR* certificateAuthority);
+}
+
+/// A list of client certificate issuer chains.
+[uuid(4271275e-9107-11eb-a8b3-0242ac130003), object, pointer_default(unique)]
+interface ICoreWebView2ClientCertificateIssuerChainList : IUnknown {
+  /// The number of issuer chains contained in
+  /// ICoreWebView2ClientCertificate.
+  [propget] HRESULT Count([out, retval] UINT* count);
+
+  /// Gets encoded data of the issuer chain at the given index.
+  /// Index 0 contains the certificate followed by intermediate CA certificates.
+  HRESULT GetValueAtIndex([in] UINT index, [out, retval] LPWSTR* issuer);
+}
+
+/// A list of client certificate object.
+[uuid(aedb012a-8e92-11eb-8dcd-0242ac130003), object, pointer_default(unique)]
+interface ICoreWebView2ClientCertificateList : IUnknown {
+  /// The number of client certificates contained in the ICoreWebView2ClientCertificateList.
+  [propget] HRESULT Count([out, retval] UINT* count);
+  /// Gets the certificate object at the given index.
+  HRESULT GetValueAtIndex([in] UINT index,
+                          [out, retval] ICoreWebView2ClientCertificate** certificate);
+}
+
+/// Add an event handler for the `ClientCertificateRequested` event.
+[uuid(c403d464-8e92-11eb-8dcd-0242ac130003), object, pointer_default(unique)]
+interface ICoreWebView2ClientCertificateRequestedEventHandler : IUnknown {
+  /// Provides the event args for the corresponding event.
+  HRESULT Invoke([in] ICoreWebView2* sender,
+                 [in] ICoreWebView2ClientCertificateRequestedEventArgs* args);
+}
+
+/// Event args for the `ClientCertificateRequested` event.
+[uuid(cb10ae1c-8e92-11eb-8dcd-0242ac130003), object, pointer_default(unique)]
+interface ICoreWebView2ClientCertificateRequestedEventArgs : IUnknown {
+  /// Host name of the server that requested client certificate authentication.
+  [propget] HRESULT Host([out, retval] LPWSTR * host);
+
+  /// Port of the server that requested client certificate authentication.
+  [propget] HRESULT Port([out, retval] int* port);
+
+  /// Returns true if the server that issued this request is an http proxy.
+  /// Returns false if the server is the origin server.
+  [propget] HRESULT IsProxy([out, retval] BOOL* isProxy);
+
+  /// Returns the `ICoreWebView2CertificateAuthorityList`.
+  /// The list contains distinguished name of certificate authorities
+  /// allowed by the server.
+  [propget] HRESULT CertificateAuthorities([out, retval]
```

Comment:
According to the comment, this isn't any old list of certificate authorities, but the list of allowed CAs. Should this be `AllowedCertificateAuthorities`?

Reply:
Yes please update